Sources/Storyboard/StoryboardSceneBased.swift

```diff
@@ -42,8 +42,8 @@ public extension StoryboardSceneBased where Self: UIViewController {
    - returns: instance of the conforming ViewController
    */
   static func instantiate() -> Self {
-    let storyboard = Self().storyboard
-    guard let vc = storyboard?.instantiateViewController(withIdentifier: self.sceneIdentifier) as? Self else {
```

Comment:
Exactly. Removing Extra `()` solving problem. However compiler is raising an issue when you trying to call self.storyboard from instantiate() function (again it's pointing to UIViewController.storyboard instead to StoryboardBased.storyboard).
Its possible to add default implementation for StoryboardSceneBased like this:
```
public extension StoryboardSceneBased {
  static var sceneIdentifier: String {
    return String(describing: self)
  }
  static var storyboard: UIStoryboard {
    return UIStoryboard(name: "foo", bundle: nil)
  }
}
```
 to switch Swift compiler to use dedicated static method, but it's dumb workaround.

After fail of all my tries with 'self' combinations to pass compilation I proposed to rename property name.

 This issue also affects example attached to library (instantiate Details View Controller after tapping Details button in InfoCell 0 or 1)